scania/corporate-ui-cms
# Improvement/restructure tab

Update content structure
- Added tab & article component
- Use dynamic zone to load component in Menu content type

How to test: 
- Check in Menu content type
- Add content with tab and without tab
- test together with https://github.com/scania/corporate-ui-site/pull/71
